Cargo.toml

```diff
@@ -89,6 +90,10 @@ sha1 = { workspace = true }
 bumpalo = { workspace = true }
 thiserror = "1.0.69"
 
+[patch.crates-io]
+# So the chia fuzzing crate can be used in the workspace
+clvmr = { path = "." }
```

Comment:
this looks suspicious. why can't we use the crates.io version?

Reply:
clvmr, a dependency of chia-fuzzing was a different version then the currently being developed version / the local source version so you would be unable to run the fuzzing tests